.travis.yml

```diff
@@ -18,8 +18,9 @@ jobs:
       deploy:
           on:
             branch: master
-            tags: true
+            env:
+            - TRAVIS_COMMIT_MESSAGE = $TRAVIS_COMMIT_MESSAGE
```

Comment:
Is this needed now? Could you just pass `$TRAVIS_COMMIT_MESSAGE` direct to lerna:publish?

Reply:
updated this